src/core/Akka/IO/TcpListener.cs

```diff
@@ -155,7 +196,17 @@ private IEnumerable<SocketAsyncEventArgs> Accept(int limit)
                 _bind.Options.ForEach(x => x.BeforeServerSocketBind(_socket));
                 _socket.Bind(_bind.LocalAddress);
                 _socket.Listen(_bind.Backlog);
-                _saeas = Accept(_acceptLimit).ToArray();
+
```

Comment:
Allocate the pool and start accepting connections on it - in theory we could do this in a single loop but this code only ever runs once.